tests/phpunit/api/v4/Action/BasicActionsTest.php

```diff
@@ -349,12 +349,45 @@ public function testContainsOperators() {
     $this->assertCount(1, $result);
     $this->assertEquals('one', $result->first()['group']);
 
+    $result = MockBasicEntity::get()
```

Comment:
Suspect it is this that is failing.

not ok 24 - Failure: api\v4\Action\BasicActionsTest::testContainsOperators
  ---
  message: Failed asserting that actual size 0 matches expected size 1.
  severity: fail

Reply:
I updated the PR and hopefully test will pass now, but currently all test are broken. Can I restart the tests somehow?